packages/tailwind/src/kz-spacing.ts

```diff
@@ -1,14 +1,41 @@
 import { heartTheme } from "@kaizen/design-tokens"
 
+const tokenDenyList: string[] = [
```

Comment:
Let's write something to record why this decision was made 😉

Reply:
Will whip up a doc in the confluence section

Reply:
Documented [here](https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3213164830/04+06+23+-+Tailwind+config+width+class+issue) 👍